Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM golang:1.25-alpine3.22 AS build
+FROM --platform=$BUILDPLATFORM golang:1.25.1-alpine3.22 AS build
```

Comment:
I would not pin the container version... as it normally is updated automatically to the latest available patch version